omnibus-sepolia-carina.toml

```diff
@@ -0,0 +1,436 @@
+depends = ["provision.system"]
+
+# wstETH/USD
+[invoke.register_node_wsteth_oracle_node]
+target = ["system.oracle_manager.Proxy"]
+func = "registerNode"
+args = [
+    # reducer node
+    1,
+    # mulDecimal
+    "<%= defaultAbiCoder.encode(['uint256'], [7]) %>",
+    # stETH/USD, wstETH/stETH
+    ["<%= extras.steth_oracle_node_id %>", "<%= extras.wsteth_to_steth_ratio_oracle_id %>"],
+]
+extra.wsteth_oracle_node_id.event = "NodeRegistered"
+extra.wsteth_oracle_node_id.arg = 0
+
+# --- Configure bfp --- #
+#
+#
+
+# --- Set pyth --- #
+#
+#
+#
+[invoke.set_pyth]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setPyth"
+args = ["<%= settings.pyth_address %>"]
+
+# --- Add Rewards distributors --- #
+#
+#
+
+[invoke.create_wsteth_rewards_distributor]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "createRewardDistributor"
+extra.wsteth_rewards_distributor.event = "RewardDistributorCreated"
+extra.wsteth_rewards_distributor.arg = 0
+
+[[invoke.create_wsteth_rewards_distributor.args]]
+poolId = "<%= settings.sc_pool_id %>"
+name = "wsteth RewardDistributor"
+token = "<%= settings.wsteth_address %>"
+collateralTypes = [
+    "<%= settings.wsteth_address %>",
+    "<%= settings.weth_address %>",
+    "<%= settings.snx_address %>",
+]
+
+[invoke.create_weth_rewards_distributor]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "createRewardDistributor"
+extra.weth_rewards_distributor.event = "RewardDistributorCreated"
+extra.weth_rewards_distributor.arg = 0
+
+[[invoke.create_weth_rewards_distributor.args]]
+poolId = "<%= settings.sc_pool_id %>"
+name = "weth RewardDistributor"
+token = "<%= settings.weth_address %>"
+collateralTypes = [
+    "<%= settings.wsteth_address %>",
+    "<%= settings.weth_address %>",
+    "<%= settings.snx_address %>",
+]
+
+# --- Global Market configuration --- #
+#
+#
+
+[invoke.set_global_market_configuration]
+target = ["bfp_market_factory.BfpMarketProxy"]
+fromCall.func = "owner"
+func = "setMarketConfiguration"
+
+[[invoke.set_global_market_configuration.args]]
+pythPublishTimeMin = 12
+pythPublishTimeMax = 60
+minOrderAge = 12
+maxOrderAge = 60
+minKeeperFeeUsd = "<%= parseEther('15') %>"
+maxKeeperFeeUsd = "<%= parseEther('500') %>"
+keeperProfitMarginUsd = "<%= parseEther('20') %>"
+keeperProfitMarginPercent = "<%= parseEther('0.2') %>"
+keeperSettlementGasUnits = 1200000
+keeperCancellationGasUnits = 600000
+keeperFlagGasUnits = 1200000
+keeperLiquidateMarginGasUnits = 1200000
+keeperLiquidationGasUnits = 1200000
+keeperLiquidationFeeUsd = "<%= parseEther('5') %>"
+keeperLiquidationEndorsed = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9" # TODO, this is currently owner, change to a keeper
+collateralDiscountScalar = "<%= parseEther('1') %>"
+minCollateralDiscount = "<%= parseEther('0.01') %>"
+maxCollateralDiscount = "<%= parseEther('0.05') %>"
+sellExactInMaxSlippagePercent = "<%= parseEther('0.05') %>"
```

Comment:
Can be removed after https://github.com/Synthetixio/synthetix-v3/pull/2146/files is merged and released